src/thirdparty/build_ui.sh

```diff
@@ -32,7 +32,8 @@ fi
 if [ ! -d $CATAPULT_HOME ]; then
   echo "setting up catapult"
   # Clone the catapult repository.
-  git clone https://github.com/catapult-project/catapult.git $CATAPULT_HOME
+  git config --global http.postBuffer 524288000 # avoid SSL read error -36
```

Comment:
It seems a bit sketchy to mess with people's git config. It might be preferable to just allow the occasional Travis failures instead of doing this. @pcmoritz what do you think?

At least, there's no need for `--global`, right?

Reply:
Yeah, I agree we shouldn't change people's git config. We could just upload a snapshot of catapult somewhere and download it with wget and good timeout settings.

Reply:
Hm you need --global since `clone` is a global operation. I could also try git clone --shallow, that might reduce the data transmitted.